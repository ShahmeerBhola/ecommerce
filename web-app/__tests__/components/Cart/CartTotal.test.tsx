import React from 'react';
import renderer from 'react-test-renderer';
import CartTotal from '../../../components/Cart/CartTotal';

describe('CartTotal component', () => {
  test('it renders properly', () => {
    const tree = renderer.create(<CartTotal />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
