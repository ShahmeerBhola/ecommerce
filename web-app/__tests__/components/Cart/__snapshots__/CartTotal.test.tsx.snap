// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartTotal component it renders properly 1`] = `
<p
  className="font-secondary font-normal text-gray-100 text-base"
>
  Total  
  <p
    className="font-secondary font-medium text-red-100 text-lg"
  >
    $0.00
  </p>
</p>
`;
